lbry/dht/protocol/iterative_find.py

```diff
@@ -163,12 +166,17 @@ async def _send_probe(self, peer: 'KademliaPeer'):
         except asyncio.TimeoutError:
             self._reset_closest(peer)
             return
+        except asyncio.CancelledError:
+            log.debug("%s[%x] cancelled probe",
+                      type(self).__name__, id(self))
+            return
```

Comment:
Can you please re-raise? Catching CancelledError can prevent it from propagating and properly cancelling other parts. [In almost all situations it must be re-raised.](https://docs.python.org/3/library/asyncio-exceptions.html#asyncio.CancelledError).

Reply:
Will do.